tests/_models/test_package.py

```diff
@@ -26,11 +27,13 @@ def test_guess_version_setuptools(mocker: MockerFixture) -> None:
     assert result == "?"
 
 
-def test_package_as_frozen_repr(tmp_path: Path, monkeypatch: pytest.MonkeyPatch) -> None:
+def test_package_as_frozen_repr(tmp_path: Path, mocker: MockerFixture) -> None:
     file_path = tmp_path / "foo.egg-link"
     with Path(file_path).open("w") as f:
         f.write("/A/B/foo")
-    monkeypatch.syspath_prepend(str(tmp_path))
+    mock_path = sys.path.copy()
+    mock_path.append(str(tmp_path))
```

Comment:
We need to keep the original part of `sys.path` because `as_frozen_repr` API need them.